docgen/devServer.js

```diff
@@ -14,6 +14,7 @@ export default function() {
   const compiler = webpack(webpackConfig);
   const bs = browserSync.create();
   bs.init({
+    ghostMode: false,
```

Comment:
This disable the mobile/desktop scroll sync, was triggering bad behavior for now